demo/site/src/documents/links/Link.tsx

```diff
@@ -47,6 +47,13 @@ export async function Link({ pageTreeNodeId }: Props): Promise<JSX.Element> {
                 redirect(link);
             }
         }
+
+        if (content.block?.type === "damFileDownload") {
+            const link = (content.block.props as DamFileDownloadLinkBlockData).file?.fileUrl.replace("/download", "");
```

Comment:
This currently redirects to the API URL. However, in newer projects we have a rewrite for DAM files in the site to support site-specific URLs for assets: https://github.com/vivid-planet/comet/blob/main/demo/site/src/middleware/damRewrite.ts. We should have this in Demo as well. Could you please add support for relative DAM URLs to Demo? https://github.com/vivid-planet/comet-starter/blob/main/site/src/util/graphQLClient.ts#L31